package/obs/rmt.conf

```diff
@@ -7,6 +7,14 @@ database:
   encoding: utf8
   timeout: 5000
   pool: 5
+  # set sslverify to true to use tls over ssl mysql connection
+  sslverify: false
+  sslkey: /some/path
+  sslcert: /some/path
+  sslca: /some/path
```

Comment:
Do we know why/when this was added?

Reply:
This is a remnant of testing you did about the mysql connection. We should drop it!